__tests__/app.test.js

```diff
@@ -115,6 +115,103 @@ describe('GET /api/articles', () => {
             }
         });
     });
+    test('returns array of objects with correct properties in given order', () => {
+        return request(app)
+        .get('/api/articles?order=asc')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles;
+
+            expect(Array.isArray(articles)).toBe(true);
+
+            expect(articles.length > 0).toBe(true);
+
+            expect(articles).toBeSortedBy('created_at', {descending: false});
+
+            for(article of articles) {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                });
+            }
```

Comment:
This should not be here, we have already asserted for this in the above test this is simply redundant.